doc/conf.py

```diff
@@ -162,3 +163,15 @@ def linkcode_resolve(domain, info):
 autodoc_default_options = {
     "show-inheritance": True,
 }
+
+# -- Sphinx-Gallery---------------
```

Comment:
If we add `autosummary_generate = True` below the sphinx gallery config (doesn't have to be there, but that's cleanest), we should get mini-galleries in the API reference pointing to examples where an object is used. See https://sphinx-gallery.github.io/stable/configuration.html#references-to-examples and e.g. https://pyxem.readthedocs.io/en/stable/reference/generated/pyxem.signals.DiffractionVectors2D.html